packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -270,3 +270,16 @@
 	top: 50%;
 	left: 50%;
 }
+
+.zoom-out-toolbar {
+
+	.components-button:focus::before {
+		box-shadow: 0 0 0 var(--wp-admin-border-width-focus) var(--wp-admin-theme-color);
+	}
+
+	.components-toolbar,
+	.components-toolbar-group {
+		background: none;
+		border: none;
+	}
```

Comment:
👋  Hi there, and sorry for just casually popping by while this PR has been open for quite some time already!

Just wanted to flag that `.components-*` class names are intended to be private implementation details, and that style overrides can lead to unexpected breakages if the component updates its internals.

Can we avoid using those style overrides? Are those changes something that could be applied directly at the component level?

Reply:
well, ideally I think these changes should go on the component, since here we are using the `unstyled` version of the toolbar, these backgrounds/border/shadow are being applied by the original component and in my opinion they shouldn't when the `unstyled` attr is present. @mirka advised [against](https://github.com/WordPress/gutenberg/pull/60123#discussion_r1603589830) the initial change, so when we found the other problems, we decided to add our own class to the toolbar instead

Reply:
I think we can go without some of this CSS, but I'm unsure about the focus, I'll get back to you

Reply:
I see! Thank you for the context, I missed it initially.

A good improvement, as also suggested by Lena in the comment you link, could still be to pass custom classnames to the component. That way, you can use those custom classnames (instead of private `.components-*` classnames) to apply the style overrides. Not ideal, but an improvement nonetheless. Would you be able to do that?

Thank you!

Reply:
Just seen https://github.com/WordPress/gutenberg/pull/60123/commits/b690f1527b580e09245e899eac3bd9297d5ef6de — thank you again!

Reply:
I still think it would be a good idea to revisit if we should make this change in the component instead. But that can happen on a different PR. Thanks for the review! If you don't have any other issues with the component changes on this one, I will merge it soon

Reply:
Agreed — and nope, no further comments from my side 🚀